docs/2-Interviews/preparing_for_interviews.md

```diff
@@ -12,22 +12,64 @@ Be prepared to:
 - talk about and work towards writing an algorithm to solve a problem, during the interview
 - you can also expect a “tell me about yourself” question (so be prepared!) and some time where they tell you about their company
 
+### What material should I know?
+
+This is a difficult question to answer. The more you know, the fewer the number of weak points that could trip you up. Knowing more things is a good thing in general, you shouldn't learn things just for interviews.
+
+That being said, you have limited time to learn things so you probably want the biggest bang for the buck.
```

Comment:
That's a great tip, I'll add that in. I can see that happening especially with culture of cramming for exam and overpreparing due to competitiveness for high grades, which doesn't matter as much for interviews because we're not evaluated in only one dimension.